### PR TITLE
fix getbalance and return all assets

### PR DESCRIPTION
**Pull request by diminator**

### CODE REVIEW

File: src/lib/bigchaindb_ledger_plugin.js [hunk 1]
```diff
@@ -99,13 +99,19 @@ class BigchainDBLedgerPlugin extends EventEmitter2 {
...
+                }
+            });
         } catch (e) {
+            console.error(e);
             throw new Error('Unable to determine current balance');
         }
 
-        if (res && res.assets && res.assets.bigchain && res.assets.bigchain.length) {
-            return res.assets.bigchain.length;
+        if (res && res.assets && res.assets.bigchain && Array.isArray(res.assets.bigchain)) {
```
Comment (sohkai) [inline src/lib/bigchaindb_ledger_plugin.js:113]: Should this instead be
`res && res.assets && Array.isArray(res.assets.bigchain) && res.assets.bigchain.length`?

Comment (diminator) [inline src/lib/bigchaindb_ledger_plugin.js:113]: why `res.assets.bigchain.length`?

**Comment by sohkai** [comment]
Is it possible to squash those empty merge commits?

**Comment by diminator** [comment]
> Is it possible to squash those empty merge commits?

you mean using rebase?

**Comment by sohkai** [comment]
Yup, with interactive mode.

File: src/lib/bigchaindb_ledger_plugin.js (hunk 1 above)
Comment (sohkai) [inline src/lib/bigchaindb_ledger_plugin.js:113]: ¯_(ツ)_/¯ It was there before, so not sure if there was an intention to error if the array is empty.

Comment (diminator) [inline src/lib/bigchaindb_ledger_plugin.js:113]: yeah, that was part of the malfunctioning behavior

Comment (sohkai) [inline src/lib/bigchaindb_ledger_plugin.js:113]: 👍

**Pull request merged**